stdlib/build.rs

```diff
@@ -26,11 +25,11 @@ fn main() -> Result<()> {
     let manifest_dir = env!("CARGO_MANIFEST_DIR");
     let asm_dir = Path::new(manifest_dir).join(ASM_DIR_PATH);
 
-    let source_manager = Arc::new(assembly::DefaultSourceManager::default());
+    let assembler = Assembler::default().with_debug_mode(cfg!(feature = "with-debug-info"));
     let namespace = "std".parse::<LibraryNamespace>().expect("invalid base namespace");
     // TODO: Add version to `Library`
-    let _version = env!("CARGO_PKG_VERSION").parse::<Version>().expect("invalid cargo version");
-    let stdlib = Library::from_dir(asm_dir, namespace, source_manager)?;
+    //let version = env!("CARGO_PKG_VERSION").parse::<Version>().expect("invalid cargo version");
```

Comment:
It would be better to remove this line - merging commented out code is a bad practice IMO. We can use the git history instead if we want to remember how to fetch the version